### PR TITLE
Adding new fancy feature

### PR DESCRIPTION
**Pull request by MorrisJobke**
- sdanfkljdsa

@PVince81 Do you want review?

@butonic Review time? It's awesome!

### CODE REVIEW

File: lib/private/previewmanager.php [hunk 1]
```diff
@@ -339,4 +339,8 @@ protected function registerCoreProviders() {
 			}
 		}
 	}
+
+	public function myNewFunction() {
+		return ['abasd'];
```
Comment (MorrisJobke) [inline lib/private/previewmanager.php]: Bad!

**Comment by MorrisJobke** [comment]
:-1:

**Comment by felixboehm** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1: Merge it! I love it!

**Comment by raghunayyar** [comment]
awesome, so cool. :D :+1:

**Comment by PVince81** [comment]
please rebase

File: lib/private/previewmanager.php [hunk 2]
```diff
@@ -339,4 +339,8 @@ protected function registerCoreProviders() {
 			}
 		}
 	}
+
+	public function myNewFunction() {
```
Comment (Xenopathic) [inline lib/private/previewmanager.php]: Please make static

**Comment by Xenopathic** [comment]
@owncloud-bot retest this please

File: lib/private/previewmanager.php (hunk 2 above)
Comment (MorrisJobke) [inline lib/private/previewmanager.php]: No

**Comment by rullzer** [comment]
Can we have unit tests for this?

**Comment by blizzz** [comment]
Can we backport this?

**Comment by Xenopathic** [comment]
cc @karlitschek

**Comment by karlitschek** [comment]
no backport This is a new feature and not a critical bugfix ;-)

**Pull request closed without merging**